### PR TITLE
Convert log level names to lowercase.

### PR DESCRIPTION
**Pull request by Jelle-S**

### CODE REVIEW

File: Resources/config/services.yml [hunk 1]
```diff
@@ -27,6 +27,11 @@ services:
         tags:
             - { name: monolog.processor }
 
+    digipolis_syslog.processor.lowercase_level:
+        class: DigipolisGent\SyslogBundle\Monolog\Processor\UidProcessor
```
Comment (dubgeiser) [inline Resources/config/services.yml]: lowercase_level: UidProcessor  One of them seems wrong

File: Monolog/Processor/LowerCaseLevelNameProcessor.php [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+namespace DigipolisGent\SyslogBundle\Monolog\Processor;
+
+/**
+ * Processor that adds the lowercase level name to a log record.
+ */
+class LowerCaseLevelNameProcessor
+{
+
+    /**
+     * Adds the lowercase level name to the record.
```
Comment (dubgeiser) [inline Monolog/Processor/LowerCaseLevelNameProcessor.php]: Documentation: it doesn't actually _add_ the level name to the record, it transforms it.

File: Monolog/Processor/LowerCaseLevelNameProcessor.php [hunk 3]
```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace DigipolisGent\SyslogBundle\Monolog\Processor;
+
+/**
+ * Processor that adds the lowercase level name to a log record.
```
Comment (dubgeiser) [inline Monolog/Processor/LowerCaseLevelNameProcessor.php]: add => transform (see comment in the __invoke() method)

**Review by daften — APPROVED** [review]

**Review by dubgeiser — CHANGES_REQUESTED** [review]
Change the docs to what the real responsibility of the class is (transformation, not adding

**Review by dubgeiser — APPROVED** [review]

**Pull request merged**